agimus_controller/main/main_hpp_panda_mpc_buffer.py

```diff
@@ -13,6 +16,13 @@
     nq = 7
     nv = 7
     pandawrapper = PandaWrapper(auto_col=True)
+    current_dir_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
In the future please use [pathlib](https://docs.python.org/3/library/pathlib.html)